libs/lmdb/Makefile

```diff
@@ -1,13 +1,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lmdb
-PKG_VERSION:=0.9.29
+PKG_VERSION:=0.9.32
 PKG_RELEASE:=1
 
-PKG_SOURCE:=LMDB_$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/LMDB/lmdb/tar.gz/LMDB_$(PKG_VERSION)?
-PKG_HASH:=22054926b426c66d8f2bc22071365df6e35f3aacf19ad943bc6167d4cae3bebb
-PKG_BUILD_DIR:=$(BUILD_DIR)/lmdb-LMDB_$(PKG_VERSION)
+PKG_SOURCE:=openldap-LMDB_$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://git.openldap.org/openldap/openldap/-/archive/LMDB_$(PKG_VERSION)
```

Comment:
change to PKG_SOURCE_PROTO:=git if no proper tarball is available.

Reply:
Done